docs/book/src/reference/labels_and_annotations.md

```diff
@@ -0,0 +1,48 @@
+**Supported Labels:**
+
+
+|label     |note     |
+|:--------:|:--------:|
+| cluster.x-k8s.io/cluster-name| It is set on machines linked to a cluster and external objects(bootstrap and infrastructure providers). |
+| topology.cluster.x-k8s.io/owned| It is set on all the object which are managed as part of a ClusterTopology. |
+|topology.cluster.x-k8s.io/deployment-name | It is set on the generated MachineDeployment objects to track the name of the MachineDeployment topology it represents. |
+| cluster.x-k8s.io/provider| It is set on components in the provider manifest. The label allows one to easily identify all the components belonging to a provider. The clusterctl tool uses this label for implementing provider's lifecycle operations. |
+| cluster.x-k8s.io/watch-filter | It can be applied to any Cluster API object. Controllers which allow for selective reconciliation may check this label and proceed with reconciliation of the object only if this label and a configured value is present. |
+| cluster.x-k8s.io/interruptible| It is used to mark the nodes that run on interruptible instances. |
+|cluster.x-k8s.io/control-plane | It is set on machines or related objects that are part of a control plane. |
+| cluster.x-k8s.io/set-name| It is set on machines if they're controlled by MachineSet. |
+| cluster.x-k8s.io/deployment-name| It is set on machines if they're controlled by a MachineDeployment. |
+| machine-template-hash| It is applied to Machines in a MachineDeployment containing the hash of the template. |
+<br>
+
+
+**Supported Annotations:**
+
+|annotation     | note     |
+|:--------:|:--------:|
+| topology.cluster.x-k8s.io/managed-field-paths   |  It is used to store the list of paths managed by the topology controller. Changes to those paths will be considered authoritative.  <br> NOTE: Managed field depends on the last reconciliation of a managed object; this list can change during the lifecycle of an object, depending on how the corresponding template + patch/variable changes over time. <br> NOTE: The topology controller is only concerned about managed paths in the spec, given that we are dropping spec. From the result to reduce verbosity of the generated annotation.<br> NOTE: Managed paths are relevant only for unstructured objects where it is not possible to easily discover which fields have been set by templates + patches/variables at a given reconcile. Instead, it is not necessary to store managed paths for typed objets (e.g. Cluster, MachineDeployments) given that the topology controller explicitly sets a well-known, immutable list of fields at every reconcile.<br> Deprecated: Topology controller is now using server side apply and this annotation will be removed in a future release.  |
```

Comment:
@fabriziopandini Should we drop this annotation from the docs?

As of v1.2 we are not using it anymore and it's hard to understand.

I would argue documenting it is more confusing to users than that it helps them in some way.